bin/alluxio

```diff
@@ -258,6 +258,11 @@ function main {
   ;;
   "runTest")
     CLASS="alluxio.cli.TestRunner"
+    ALLUXIO_MASTER_HOSTNAME=$(${BIN}/alluxio getConf alluxio.master.hostname)
```

Comment:
I think it is confusing that we are using the same environment variable name here as for the public API ... mark this variable as local (https://google.github.io/styleguide/shell.xml#Use_Local_Variables) if it is indeed only used locally